### DISCUSSION TITLE
tflint --chdir option does not work?

### DISCUSSION THREAD

**Discussion by yannik1999** [oxsecurity/megalinter#2467] [Q&A]
I want to change to a specific directory when using tflint. To do that, I use this in the megalint config:
TERRAFORM_TFLINT_ARGUMENTS: "--chdir=/tmp/lint/environments/production/application"

But when I do that, I just get an error:
Cannot use --chdir and directory argument at the same time. 

I think this is related to some new versions in tflint (release 0.45). 
https://github.com/terraform-linters/tflint/pull/1678/commits

---

I'm using Megalinter in Azure Pipelines. 

MegaLinter Version:
v6

TFLint Version
0.45.0

Terraform Version:
- 1.4.2

Operating System
-  Linux

**Comment by nvuillam** [comment]
@yannik1999 did you try with `LOG_LEVEL: DEBUG` to see the exact call to tflint ?

**Reply by yannik1999** [reply]
I tried that now. The input for the previous steps gets much more verbose. But the error message has not more output:
![image](https://user-images.githubusercontent.com/105272469/230331079-2739ddd6-e6b2-4b0a-a969-365277251b5f.png)

**Reply by nvuillam** [reply]
Do you see the tflint call ?
Please can you paste it here ?
